packages/2019-disaster-game/src/fonts.css

```diff
@@ -1,2 +1,3 @@
 /* Boggaloo for Game */
 @import url("https://fonts.googleapis.com/css?family=Boogaloo&display=swap");
+@import url("https://fonts.googleapis.com/css?family=Luckiest+Guy&display=swap");
```

Comment:
Need to make sure these get loaded for offline mode somehow

Reply:
Yup. Was thinking about that

Reply:
^^^ @DingoEatingFuzz heads up ^^^